activity/activity.go

```diff
@@ -41,6 +41,10 @@ type (
 
 	// RegisterOptions consists of options for registering an activity
 	RegisterOptions = internal.RegisterActivityOptions
+
+	// ExtraInformation is an additional data which should be sent to the server to influence its decision
+	// about the next retry
+	ExtraInformation = internal.ActivityExtraInformation
```

Comment:
"extra information" implies that it's something like the existing "GetInfo" (also, it's confusing to call it "information" here when we use "info" elsewhere). I think "extra attributes" is probably best. But we can wait until we see what happens in https://github.com/temporalio/api/pull/334/files#r1422999174. There are a lot of different words like "extra", "information", "requests", etc being used.